seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2522,11 +2522,11 @@
         "(function() { return logOutput || \"console.log never called\"; })();"
       ],
       "solutions": [
-        "function myLocalScope() {\n  'use strict';\n  \n  var myVar;\n  console.log(myVar);\n}\nmyLocalScope();"
+        "function myLocalScope() {\n  'use strict';\n  \n  var myVar = 100;\n  console.log(myVar);\n}\nmyLocalScope();"
       ],
       "tests": [
         "assert(typeof myVar === 'undefined', 'message: No global <code>myVar</code> variable');",
-        "assert(/var\\s+myVar/.test(code), 'message: Add a local <code>myVar</code> variable');"
+        "assert(/^[^\\/\\n]*var[^\\S\\n]+myVar[^\\S\\n]*=[^\\S\\n]*100[^\\S\\n]*;?.*$/m.test(code), 'message: Add a local <code>myVar</code> variable with a value of <code>100</code>');"
```

Comment:
This does indeed work, though I was a bit confused as to which console.log line to update. I also am not a fan of the "follow the directions in the code" part. I feel like we need to figure out a more elegant way of conveying this.